stable/yugaware/templates/configs.yaml

```diff
@@ -347,6 +347,10 @@ data:
             regex: "(.*)"
             target_label: "container_name"
             replacement: "$1"
+          - source_labels: ["__name__", "unit"]
```

Comment:
Am I understand correctly that the recent version of kube-state-metrics is having kube_pod_container_resource_requests with core label in it, instead of the old  kube_pod_container_resource_requests_cpu_cores?

If so - why we're trying to convert to the older version instead of converting to the newer version and having recent metric name in Platform code? I think we should prefer recent version of metric names, where possible.

Reply:
@anmalysh-yb 
> Am I understand correctly that the recent version of kube-state-metrics is having kube_pod_container_resource_requests with core label in it, instead of the old kube_pod_container_resource_requests_cpu_cores?

Correct.

> If so - why we're trying to convert to the older version instead of converting to the newer version and having recent metric name in Platform code? I think we should prefer recent version of metric names, where possible.

We're doing similar for few node exporter metrics.

Reply:
For node exporter metrics - we're converting older node-exporter version metrics names to the latest version metric names.
We should do the same for kube metrics, instead of converting recent version metric names to the older version.

Reply:
@anmalysh-yb I tried the suggested approach, and it's a fairly straightforward change. The only concern I have is we won't be able to see historical data because moving forward, it'll use the new metric name, and the Prometheus have historical data for the old metric. Could you please let me know your opinion on this?

Reply:
I think that's acceptable to loose historical view in Platform UI in case we do up-versions like that.
We still have data in Prometheus UI in case we need it for some troubleshooting.